pokemons/helpers.py

```diff
@@ -22,3 +22,10 @@ def is_pokemons_sum_valid(pokemons):
     pokemons_sum = sum([sum(aux[i].values()) for i in range(len(aux))])
 
     return pokemons_sum <= limit
+
+
+def exists(name):
```

Comment:
Helpers with very descriptive name is a huge plus. Instead of naming this helper `exists`, you could call it `pokemon_exists`.

Reply:
Done!